src/admin-views/help-hub/resources/common-issues.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * The template that displays the resources tab on the help page.
+ *
+ * @var Tribe__Main $main             The main common object.
+ * @var Hub         $help_hub         The Help Hub class.
+ * @var string      $template_variant The template variant, determining which template to display.
+ */
+
+use TEC\Common\Admin\Help_Hub\Hub;
+
+$section = $help_hub->handle_resource_sections();
+
+if ( empty( $section['common_issues'] ) ) {
+	return;
+}
+?>
+
+<div class="tec-settings-form__content-section">
+	<div class="tec-settings-form__header-block">
+		<h3 class="tec-settings-form__section-header tec-settings-form__section-header--sub">
+			<?php echo esc_html_x( 'Common issues', 'Common issues section title', 'tribe-common' ); ?>
+		</h3>
+		<p class="tec-settings-form__section-description">
+			<?php echo esc_html_x( 'Having trouble? Find solutions to common issues or ask our AI Chatbot.', 'Common issues section paragraph', 'tribe-common' ); ?>
```

Comment:
```suggestion
			<?php echo esc_html_x( 'Having trouble? Find solutions to common issues or ask our AI Chatbot.', 'Common issues section description', 'tribe-common' ); ?>
```

Remember - this is supposed to add _context_ for the translator so not just _what_ but _why_.